src/components/Login.vue

```diff
@@ -5,41 +5,129 @@
+                </div>
+            </div>
+        </div>
 
-          <button @click="login" class="button">Log In</button> -->
+        <!-- DRAG -->
+        <div class="container-fluid plan p-0">
+            <div class="drag row no-gutters">
+                <div class = "col" >
+                    <img id= "hide" style="position:relative" class="preview" src="../assets/images/drag.svg" alt = "Dragging preview" />
+                </div>
+                <div class = "col comment" >
+                    <h1  class="head" style = "text-align: left;">Drag Your Course In</h1>
+                    <p class= "sub"> Courseplan’s intuitive interface recommends courses based on unfulfilled
+                        requirements and allows you to easily drag and drop them into your planner </p>
+                </div>
+            </div>
+        </div>
 
-          <!-- <div class="extras">
-          <a>Forgot Password</a>
-          <a>Create an Account</a>
-        </div> -->
-        </form>
-      </div>
-    </section>
+
+        <!-- SEMESTERS -->
+        <div class="container-fluid p-0 small">
+            <div class="row semester m-0">
+                <div class = "col-4 comment" >
+                    <h1 class= "head" style = "text-align: left;">Plan Your Semesters</h1>
+                    <p class= "sub">Use Courseplan’s semesterly planner to choose courses well in advance and ensure that you never miss a requirement</p>
+                </div>
+                <div class = "col" >
+                    <img id= "hide" class="schedule" src="../assets/images/schedule.svg" alt = "Plan preview" />
+                </div>
+            </div>
+        </div>
+
+        <!-- FIRST -->
+        <div class="container first m-0">
+          <div class = "container inside">
+              <div class="row justify-content-center">
+                <div class= "col justify-content-center">
+                  <h1 class= "head-center text-center">Be The First One To Use It</h1>
+                  <p class= "sub text-center" style= "padding: 30px; ">Gain early access by filling out your email below and help us grow into what you need!</p>
+                </div>
+              </div>
+              <div class="row email ">
+                  <div class="col-7  ">
+                      <input type="text" placeholder="Your Email Address" v-model="newUser.email" >
+                  </div>
+                  <div class="col-5 ">
+                      <button class="email-button" variant= "primary"  v-on:click="addUser"> Add email </button>
+                  </div>
+              </div>
+            </div>
+          </div>
+
+
+        <!--FOOTER-->
+        <div class="container-fluid footer p-0">
+            <div class= "row footer justify-content-center m-0">
+                <div class="col-3">
+                    <p class= "footer"> Cornell DTI @ 2020</p>
```

Comment:
Can you make this year whatever the current year is instead of 2020 so we don't need to update this in the future?